docs/meshstack.metadata-tags.md

```diff
@@ -229,8 +106,8 @@ meshStack makes metadata available to [Landing Zones](./meshcloud.landing-zones.
 meshStack automatically derives [metadata tags](./meshcloud.metadata-tags.md) for [meshTenants](./meshcloud.tenant.md) based on the metadata tags set on the [meshProject](./meshcloud.project.md), the [payment method](./meshcloud.payment-methods.md) configured on the meshProject and
 the [meshCustomer](./meshcloud.customer.md) it belongs to.
 
-It's possible that these sources provide different values for the same tag key. For example, both the meshCustomer and
-meshProject [tag schema](#tag-schemas) could define a `cmdb-id` tag. Setting the `cmdb-id` tag value on the
+It's possible that these objects have tags with the same tag key. For example, both the meshCustomer and
+meshProject could contain a `cmdb-id` tag. Setting the `cmdb-id` tag value on the
```

Comment:
d: I like the example. We could add a table or image or something that shows the different tags at meshObject level and then the tag that is actually used for resource tagging to give a visual example in addition to the precedence rule.
e.g.
meshObject  | tag name | tag value |
meshCustomer |  cmdb-id  |   12         |
meshPaymentMethod |  cmdb-id  |   44         |
meshProject |  cmdb-id  |   79         |
result | cmdb-id | 79 |